java/ql/src/semmle/code/java/dataflow/internal/DataFlowImpl.qll

```diff
@@ -2105,6 +2130,12 @@ private module Stage4 {
     PrevStage::revFlow(node1, _, _, _, unbind(config))
   }
 
+  bindingset[node, ap]
+  private predicate filter(Node node, Ap ap) { any() }
+
+  bindingset[ap, contentType]
+  private predicate typecheckStore(Ap ap, DataFlowType contentType) { any() }
```

Comment:
Perhaps add comments that these are trivial because filtering/type checking happens in previous stage.